lib/devices/Roborock.js

```diff
@@ -271,7 +274,11 @@ class Roborock extends MiioVacuum {
      * @param {string} cron
      */
     async addTimer(cron) {
-        await this.sendLocal("set_timer", [[Date.now().toString(),[cron, ["",""]]]], {});
+        if (this.isGen3) {
+            await this.sendLocal("set_timer", [[Date.now().toString(), [cron, ["start_clean", {"fan_power": 102, "segments": "", "repeat": 1, "clean_order_mode": 1}]]]], {});
```

Comment:
Shouldn’t `fan_power` reference the current fan setting?

Reply:
I do not think so. 
You should actually be able to set it when creating the timer in the web interface, but this PR was intended as a bug fix only.